### PR TITLE
Submission version and data

### PR DESCRIPTION
**Pull request by leclere**

### CODE REVIEW

File: Philpott.jl [hunk 1]
```diff
@@ -0,0 +1,14 @@
+srand(1111)
```
Comment (frapac) [inline Philpott.jl:1]: Maybe move this file in `/run` folder?

File: res/old/1/res_cia_12_50.csv [hunk 2]
```diff
@@ -0,0 +1,4 @@
+870192.3446021639
```
Comment (frapac) [inline res/old/1/res_cia_12_50.csv:1]: Do we want to version the results?

File: src/innerapprox.jl [hunk 3]
```diff
@@ -6,7 +6,7 @@
 ################################################################################
 
 "Build inner approx LBO with value function `V` inside JuMP model `m`."
-function build_inner_approx!(m, V)
+function build_inner_approx!(m, V; Lip = LIPSCHITZ)
```
Comment (frapac) [inline src/innerapprox.jl]: I would change to `lipschitz=LIPSCHITZ`.

File: src/main.jl [hunk 4]
```diff
@@ -6,21 +6,22 @@
...
-mpts = MPTS([:FRA, :GER], 3, 10)
+NAMES = [:FRA, :GER, :ESP, :UK, :PT, :ITA, :SUI, :BEL]
+mpts = MPTS(NAMES, 12, 10)
 
 # Init SDDP interface
 sddpprimal = initprimal(mpts)
 
 if ALGO == 1
-    ubp, stdp = runprimal!(sddpprimal)
+    ubp, stdp, t = runprimal!(sddpprimal)
```
Comment (frapac) [inline src/main.jl]: `t` -> `trajectories`

**Review by frapac — CHANGES_REQUESTED** [review]
Is it possible to merge this change onto the `fp/innerapprox` branch?

Otherwise, I am ok to merge. I am just wondering whether it is worthy to version the results on this repo.

File: Philpott.jl (hunk 1 above)
Comment (leclere) [reply Philpott.jl:1]: We can. Anyway this file is not destined to stay.

File: res/old/1/res_cia_12_50.csv (hunk 2 above)
Comment (leclere) [reply res/old/1/res_cia_12_50.csv:1]: Maybe, to have them fixed and available for the next iteration of the paper.

**Pull request merged**